Lib9c/Action/IssueToken.cs

```diff
@@ -0,0 +1,98 @@
+using System.Collections.Generic;
+using System.Linq;
+using Bencodex.Types;
+using Lib9c;
+using Libplanet.Action;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+using Nekoyume.Action.Garages;
+using Nekoyume.Exceptions;
+using Nekoyume.Model.State;
+using Nekoyume.Module;
+using Nekoyume.TableData.Garages;
+
+namespace Nekoyume.Action
+{
+    [ActionType(TypeIdentifier)]
+    public class IssueToken : ActionBase
+    {
+        public const string TypeIdentifier = "issue_token";
+        public Address AvatarAddress;
+        public List<FungibleAssetValue> FungibleAssetValues;
+        public List<(int id, int count)> Items;
+
+        public IssueToken()
+        {
+        }
+
+        public override IValue PlainValue => Dictionary.Empty
+            .Add("type_id", TypeIdentifier)
+            .Add(
+                "values", Dictionary.Empty
+                    .Add("a", AvatarAddress.Serialize())
+                    .Add("f", new List(FungibleAssetValues.Select(f => f.Serialize())))
+                    .Add("i", new List(Items.Select(i => List.Empty.Add(i.id).Add(i.count))))
+            );
+
+        public override void LoadPlainValue(IValue plainValue)
+        {
+            var dict = (Dictionary)((Dictionary)plainValue)["values"];
+            AvatarAddress = dict["a"].ToAddress();
+            FungibleAssetValues = ((List)dict["f"]).ToList(f => f.ToFungibleAssetValue());
+            Items = new List<(int id, int count)>();
+            var list = (List)dict["i"];
+            foreach (var value in list)
+            {
+                var innerList = (List)value;
+                Items.Add(((Integer)innerList[0], (Integer)innerList[1]));
+            }
+        }
+
+        public override IWorld Execute(IActionContext context)
+        {
+            context.UseGas(1);
+            Addresses.CheckAvatarAddrIsContainedInAgent(context.Signer, AvatarAddress);
+            var state = context.PreviousState;
+            var sheet = state.GetSheet<LoadIntoMyGaragesCostSheet>();
+            var garageCost = sheet.GetGarageCost(FungibleAssetValues, Items);
+            state = state.TransferAsset(
+                context,
+                context.Signer,
+                Addresses.GarageWallet,
+                garageCost
+            );
+            foreach (var fungibleAssetValue in FungibleAssetValues)
+            {
+                var currency = fungibleAssetValue.Currency;
```

Comment:
It should check the currency is `NCG`, or it has `minters` field. Because, to use issued *Item Token*s, it should use `ClaimItems` action. And the `ClaimItems` action burns the wrapped currency FAVs and mints unwrapped currency FAVs. But it cannot mint the FAVs when its currency has `minters` field. For instance, `NCG` has `minters` field so `FAV__NCG` wrapped currency FAVs cannot be unwrapped.

Reply:
LoadIntoMyGaragesCostSheet already check `Currency` or `Item` can be minting.

Reply:
It seems resolved by [9071233](https://github.com/planetarium/lib9c/pull/2720/commits/90712336e508662bb52cd340876d5e89392a29ef)